src/gui/shellwidget/shellwidget.cpp

```diff
@@ -32,15 +32,19 @@ void ShellWidget::setDefaultFont()
 {
 #if defined(Q_OS_MAC)
 #  define DEFAULT_FONT "Courier New"
+#  define DEFAULT_FONT_WIDE "Courier New"
```

Comment:
Let's not expand preprocessor macros; the values are the same anyways.

You can use `DEFAULT_FONT`.

Reply:
I removed DEFAULT_FONT_WIDE, and use same font for wide by default.